### DISCUSSION TITLE
wikijs and github problem

### DISCUSSION THREAD

**Discussion by peutre** [requarks/wiki#4672] [Help / Questions]
Hello everyone, i got one wikjs V1 working well with **github ssh**.
But my **Wikijs V2** is not working for one synchronisation with an other guthub repository.

I always got the error_ _"fatal: ni ceci ni aucun de ses répertoires parents n'est un dépôt git : .git" "none of those parent directories are git depot_

![2021-11-08_10h39_54](https://user-images.githubusercontent.com/51857358/140718867-43c813f8-f095-4c69-8c69-90e69c707497.png)

```
ssh -T git@github.com
Hi peutre/wikijs2! You've successfully authenticated, but GitHub does not provide shell access.
```

I don't know what is wrong, maybe someone can help me about this case ? i will be very happy.

Thank you,

Jean

**Comment by ieaster1** [comment]
Seems like that repo needs to be initialized.  Have you confirmed you can pull/push to that repo from your computer?
